CopilotKit/examples/next-openai/src/app/helloworld/page.tsx

```diff
@@ -0,0 +1,84 @@
+"use client";
+
+import {
+  CopilotProvider,
+  useMakeCopilotActionable,
+  useMakeCopilotReadable,
+} from "@copilotkit/react-core";
+import { CopilotSidebarUIProvider } from "@copilotkit/react-ui";
+import { useState } from "react";
+
+const HelloWorld = () => {
+  return (
+    <CopilotProvider chatApiEndpoint="/api/copilotkit/chat">
```

Comment:
Oh actually one problem!

I just refactored this (that was the cause of the error we were seeing yesterday)
It should point to `/api/copilotkit/openai`